docs/algorithms/AlphaShape.asciidoc

```diff
@@ -14,7 +14,7 @@ the alpha shape.
 
 [[alphashape]]
 .Simple Alpha Shape
-image::AlphaShape.png[A simple example of an alpha shape,scalewidth="50%"]
+image::images/AlphaShape.png[A simple example of an alpha shape,scalewidth="50%"]
```

Comment:
Because the images were moved from `docs/algorithms/` to `docs/algorithms/images/` this change should look like this:
```
image:algorithms/images/AlphaShape.png[A simple example of an alpha shape,scalewidth="50%"]
```
This is repeated for all images that you moved.